deployment/keystone/changeset/configure_contracts.go

```diff
@@ -23,6 +24,7 @@ func ConfigureInitialContractsChangeset(e deployment.Environment, cfg InitialCon
 		Env:              &e,
 		RegistryChainSel: cfg.RegistryChainSel,
 		Dons:             cfg.Dons,
+		OCR3Address:      cfg.OCR3Address,
```

Comment:
I think this changeset is unused/deprecated. We don't configure multiple contracts at once any more.